src/app/src/stagecontrol.cpp

```diff
@@ -108,19 +108,54 @@ void StageControl::loadList(std::string fileName) {
     m_positions.clear();
     ui->stageLocations->clear();
 
+    auto file = toml::parse(fileName);
```

Comment:
I don't think we need this function anymore since we removed the save/load buttons from the stage control dialog

Reply:
The main window is calling it whenever the plate format drop down selection is changed. Should we load the position list another way?